app/src/main/res/values/dimens.xml

```diff
@@ -24,6 +24,15 @@
   <dimen name="grid_35">280dp</dimen>
   <dimen name="grid_40">320dp</dimen>
 
+  <!-- Kickstarter six-point grid keylines -->
```

Comment:
ok i just did a quick dry run of switching to a 6pt grid and it's not something that should block messages, since it'll take more time to phase in the new grid i'll chat with @maggieKSR about it 

for the mean time i'm okay with appending new grid values with `new_` that way when we search and replace it'll be easy to find them